src/pages/wallet/components/SettingsSection.tsx

```diff
@@ -26,6 +33,18 @@ interface SettingsSectionProps {
   formType: keyof typeof FORM_TYPE_ENUM
 }
 
+type AvailableFeeTypes = {
+  [FeeTypesEnum.Charge]: FeeTypesEnum.Charge
+  [FeeTypesEnum.Commitment]: FeeTypesEnum.Commitment
+  [FeeTypesEnum.Subscription]: FeeTypesEnum.Subscription
+}
```

Comment:
Why not simple use this notation? 
```suggestion
type AvailableFeeTypes = FeeTypesEnum.Charge | FeeTypesEnum.Commitment | FeeTypesEnum.Subscription
```

Reply:
Fair point 😄 
Will update, indeed that makes no sense